src/Umbraco.Core/Persistence/DatabaseSchemaHelper.cs

```diff
@@ -90,20 +138,54 @@ internal void CreateDatabaseSchemaDo(IMigrationEntryService migrationEntryServic
             _logger.Info<Database>("Finalized database schema creation");
         }
 
+        /// Creates a new table in the database based on the type of <typeparamref name="T"/>.
+        /// 
```

Comment:
Good idea to get rid of this blank line?

Reply:
There's also no <summary> tag here